app/src/main/java/net/gsantner/markor/activity/MainActivity.java

```diff
@@ -394,7 +395,8 @@ public void onViewPagerPageSelected(int pos) {
         (_lastBottomMenuItem != null ? _lastBottomMenuItem : menu.getItem(0)).setChecked(false);
         _lastBottomMenuItem = menu.getItem(pos).setChecked(true);
         updateFabVisibility(pos == 0);
-        _toolbar.setTitle(new String[]{_currentTitle, getString(R.string.todo), getString(R.string.quicknote), getString(R.string.linkbox), getString(R.string.more)}[pos]);
+        FilesystemViewerFragment fragment = (FilesystemViewerFragment) _viewPagerAdapter.getFragmentByTag(FilesystemViewerFragment.FRAGMENT_TAG);
```

Comment:
it may not happen often, but it is possible that FilesystemViewerFragment is null. can you fallback to _currentTitle in that case? (`frag == null ? _currentTitle : getCurrent..`)